packages/charts/react-charting/etc/react-charting.api.md

```diff
@@ -119,6 +119,14 @@ export const DataVizPalette: {
     highSuccess: string;
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33348&commitId=556bf0e69294b399cc1ae4e38dceb49f40ccb6be&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>